API/Controllers/UserController.cs

```diff
@@ -625,15 +627,9 @@ public async Task<IActionResult> GetRecommendedProjects(int amount)
                 List<Project> projectRecommendations = await userService.GetRecommendedProjects(user.Id, amount);
                 return Ok(mapper.Map<List<Project>, List<ProjectResourceResult>>(projectRecommendations));
 
-            } catch(RecommendationNotFoundException e)
+            } catch(RecommendationNotFoundException)
             {
-                ProblemDetails problem = new ProblemDetails
-                                         {
-                                             Title = "Failed getting the recommendations",
-                                             Detail = e.Message,
-                                             Instance = "948319D2-1A19-4E00-AF50-DB5D096AFD39"
-                                         };
-                return NotFound(problem);
+                return Ok(new List<Project>());
```

Comment:
I think, to keep everything consistent, you should return an empty list of ProjectResourceResult